src/instrumenter/state_var_instrumenter.ts

```diff
@@ -205,45 +202,45 @@ const assignSuffixMap: { [key: string]: string } = {
 /**
  * Get a string descriptor for a type to be used in naming conventions
  */
-function getTypeDescriptor(typ: SType): string {
-    if (typ instanceof SAddressType) {
+function getTypeDescriptor(typ: TypeNode): string {
```

Comment:
I'm wondering if we can reuse `TypeNode`s `pp()` method. There a few differences here, though. And I guess it serves a bit different purpose.

Reply:
They output differently formatted strings. TypeNode's `pp()` method outputs human-readable strings with spaces. This method outputs strings with no spaces that are used for computing wrapper names

Reply:
I don't think we can re-use that here